README.md

```diff
@@ -8,54 +8,38 @@ Android POC for www.covid-watch.org
 ## Who are we? What is this app?
 
 This repository is focused on the implementation of the Android version of the Covid Watch app. Our goals are to:
-- Allow users to anonymously record interactions with others using the [TCN Protocol](https://github.com/TCNCoalition/tcn-client-android)
+- Allow users to anonymously record interactions with others using the Google/Apple exposures notification api for Android
+    (https://github.com/google/exposure-notifications-android)
 - Notify users if someone they've interacted with in the past 2 weeks indicates they've tested positive for COVID-19 (again, anonymously!)
 - Offer a seamless UX to complete all of the above!
 
-The current version of the Figma we're working towards: https://www.figma.com/file/0uL6s79o21jwUFZz16Dr8b/Covid-Watch---App-v1.8?node-id=0%3A1
+The current version of the Figma we're working towards: (https://www.figma.com/file/I4OOjT4VhsSberDIAZTkcH/Covid-Watch---App-v2-(dev)?node-id=0%3A1
+
 
 ## Setup
 
 Clone this repo from the `develop` branch:
 
-```
 git clone git@github.com:covid19risk/covidwatch-android.git
-```
-
-We are using tracking TCN's project as a submodule for now tracking the develop branch, so don't forget to init and fetch.
-
-First time:
-
-```
-$ git submodule update --init --recursive --remote
-```
-
-To Update:
-
-```
-git submodule update --remote
-```
 
 Open the project in Android Studio. Install onto a phone of yours with the `app` configuration, and you're free to explore the app! Its optimal to install on 2 phones as much of the behavior of the app depends on 2 phones interacting.
 
 **Note:** You cannot run this app on an emulator! We are dependent on Bluetooth being on and active, and most standard Android emulators do not have Bluetooth drivers.
 
 ## Looking to contribute?
 
-- Run on your own device to explore the UX. Look at the [Figma](https://www.figma.com/file/0uL6s79o21jwUFZz16Dr8b/Covid-Watch---App-v1.8?node-id=0%3A1) for what the UX should look like. If you have any feedback/find any problems, create an issue!
-- Look at https://github.com/orgs/covid19risk/projects/1 for existing issues. If you see something you want to work on, assign yourself to it, set it to in progress, and make a PR to the `develop` branch.
+- Run on your own device to explore the UX. Look at the [Figma](https://www.figma.com/file/I4OOjT4VhsSberDIAZTkcH/Covid-Watch---App-v2-(dev)?node-id=0%3A1) for what the UX should look like. If you have any feedback/find any problems, create an issue!
+- Look at https://blog.google/documents/68/Android_Exposure_Notification_API_documentation_v1.2.pdf for existing issues. If you see something you want to work on, assign yourself to it, set it to in progress, and make a PR to the `develop` branch.
```

Comment:
That PDF doesn't show issues in our application. Please return previous item with link to the GitHub project.